readme.md

```diff
@@ -25,8 +25,23 @@ sudo apt install libz3-dev
 Make sure that you have the [`venv`](https://docs.python.org/3/library/venv.html)
 module for your python installation.
 
+### Installation Method 1:
 
-### Installation from source
+Without any virtual environment activated run the following in order:
+
+```bash
+yarn global add @nethermindeth/warp
+which warp - /home/guy/.yarn/bin/warp
+warp install
+```
+
+Test installation works with:
+
+```
+warp transpile <somecontract>
+```
+
+### Installation Method 2 (from source):
```

Comment:
We should put this in the contributing section, people interested in development will use this method
people who just want to use the compiler can/will just use the other method